src/vcpkg/postbuildlint.cpp

```diff
@@ -222,6 +238,31 @@ namespace vcpkg::PostBuildLint
         return LintStatus::SUCCESS;
     }
 
+    static LintStatus check_for_vcpkg_port_config(const Files::Filesystem& fs,
+                                                  const Build::BuildPolicies& policies,
+                                                  const fs::path& package_dir,
+                                                  const PackageSpec& spec)
+    {
+        const fs::path relative_path =
+            fs::u8path("share") / fs::u8path(spec.name()) / fs::u8path("vcpkg-port-config.cmake");
+        const fs::path absolute_path = package_dir / relative_path;
+
+        if (policies.is_enabled(BuildPolicy::CMAKE_HELPER_PORT))
+        {
+            if (!fs.exists(absolute_path))
+            {
+                System::print2(System::Color::warning,
+                               "The /",
+                               fs::u8string(relative_path),
+                               " "
+                               "file does not exist. This file must exist for CMake helper ports.\n");
```

Comment:
This line break is strange.

Reply:
that is strange...

Reply:
however, it is the way that clang-format wants to do it...